### PR TITLE
Docs(web-react, web): Add Stack demo #DS-908

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Tag demos are now same in all packages

### Issue reference

[Demo: Stack má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-908)

### CODE REVIEW

File: packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import Stack from '../Stack';
+
+const StackBlocksWithInnerDividersAndVerticalSpacing = () => (
+  <Stack elementType="ul" hasSpacing hasIntermediateDividers>
+    {[1, 2, 3].map((i) => (
```
Comment (literat) [inline packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx:7]: :-) Generating values is an interesting problem. There are multiple ways/oneliners how to, based on a single value, create an array full of values.

```js
var foo = [ ...Array(N).keys() ];
var foo = Array(N).fill().map((v,i)=>i);
Array.from({ length: N }, (v, i) => i)
```

;-)

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx (hunk 1 above)
Comment (crishpeen) [reply packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx:7]: And which one is the best? :)

File: packages/web/src/scss/components/Stack/index.html [hunk 2]
```diff
@@ -15,11 +17,15 @@ <h2 class="docs-Heading">Stacked Form Fields</h2>
...
+
+<section class="docs-Section">
 
-  <section class="docs-Section">
+  <h2 class="docs-Heading">Stacked Blocks</h2>
 
-    <h2 class="docs-Heading">Stacked Blocks</h2>
+  <div class="docs-Stack docs-Stack--stretch">
 
     <ul class="Stack">
```
Comment (crishpeen) [inline packages/web/src/scss/components/Stack/index.html:30]: 😀 stack inside stack! We need to go deeper!

**Review by crishpeen — APPROVED** [review]

File: packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx:7]: All are good! 😁

**Pull request merged**